src/_guides/language/effective-dart/design.md

```diff
@@ -1087,8 +1354,8 @@ void log(Object object) {
   print(object.toString());
```

Comment:
I'd suggest dropping the "// Accepts any object" comment.

Reply:
Done, and turned the other into a real doc comment.